components/org.wso2.carbon.identity.mgt.store.connector.jdbc/src/main/java/org/wso2/carbon/identity/mgt/store/connector/jdbc/connector/JDBCIdentityStoreConnector.java

```diff
@@ -518,6 +517,50 @@ public IdentityStoreConnectorConfig getIdentityStoreConfig() {
     }
 
     @Override
+    public List<String> getUsers(List<Attribute> attributes, int offset, int length)
+            throws IdentityStoreConnectorException {
+
+        List<String> userIdsToReturn = new ArrayList<>();
+        Map<String, String> properties = identityStoreConfig.getProperties();
+        String databaseType =  properties.get(ConnectorConstants.DATABASE_TYPE);
+        String sqlQuerryForUserAttributes;
+
+        if (databaseType != null && (databaseType.equalsIgnoreCase("MySQL") ||
```

Comment:
1. We need to have "MySQL" "H2" etc in global constant somewhere.
2. We need to have a utility to do the databaseType.equalsIgnoreCase(X)
Those are not urgent. But design considerations across all other Data accesses.

Reply:
Fixed with https://github.com/wso2-extensions/carbon-security-user-store-jdbc/pull/95